client/Lykke.Service.Balances.Client/BalancesClient.cs

```diff
@@ -116,6 +116,19 @@ public async Task<IEnumerable<ClientBalanceResponseModel>> GetTotalBalances()
             throw new Exception(UexpectedApiResponse);
         }
 
+        public async Task<ClientBalanceResponseModel> GetTotalBalance(string assetId)
```

Comment:
GetTotalBalanceAsync

Reply:
Decided to name a method without ...Async - since all other methods are also w/o it